api/app/tests/weather_models/test_env_canada_rdps.py

```diff
@@ -119,4 +120,4 @@ def test_process_rdps(mock_download,
     # All files, except one, are marked as already having been downloaded, so we expect one file to
     # be processed.
     sys.argv = ["argv", "RDPS"]
-    assert app.jobs.env_canada.process_models() == 1
+    assert app.jobs.env_canada.process_models(StationSourceEnum.LOCAL_STORAGE) == 1
```

Comment:
Do we only use `LOCAL_STORAGE` source for tests? If so, would it make sense to rename it to reflect that?

Reply:
At this point I think this is the case. We can probably rip out all the logic associated with using the WFWX API for station data and just always fetch the latest. We'd just have to mock the call for testing purposes.

Reply:
I renamed `LOCAL_STORAGE` to `TEST` in https://github.com/bcgov/wps/pull/2858/commits/7f28f715ed9e44df76e1f88904722b176eb07ad3